iabtcf-decoder/src/main/java/com/iabtcf/utils/IntIterable.java

```diff
@@ -28,7 +30,17 @@
 
     boolean contains(int value);
 
-    boolean containsAll(int... source);
+    default boolean containsAll(int... source) {
```

Comment:
can you conform to the style of the existing code base? default is superflevous and we don't need streams for a 3 line for loop.

Reply:
you told me to pull the implementation up to interface so the keyword defalut is needed. 
No value added for replacing streams with loops